README.md

```diff
@@ -25,12 +25,16 @@ two subclasses:
 2. `prpy.planning.base.MetaPlanner`: combines the output of multiple motion
    planners, each of which is a `BasePlanner` or another `MetaPlanner`
 
-Each planner has one or more *planning methods*, annotated with the
-`@PlanningMethod` decorator, that look like ordinary functions. However, unlike
-an ordinary function, calling a planning method clones the robot's environment
+Each planner has one or more *planning methods*, annotated with either the
+`@LockedPlanningMethod` or `@ClonedPlanningMethod decorator, that look like
+ordinary functions. However, unlike an ordinary function, calling a planning
+method either locks the robot environment or clones the robot's environment
 into a *planning environment* associated with the planner. Planning occurs in
-the cloned environment to allow PrPy to run multiple planners in parallel and
-to paralellize planning and execution.
+the locked or cloned environment to allow PrPy to run multiple planners in
+parallel and to paralellize planning and execution.  In general, locked
+planning methods are used for calls that will terminate extremely quickly,
+while cloned planning methods are used for calls that might take a significant
+amount of time.
```

Comment:
This description does not clearly articulate the difference between the two decorators. For example, it currently implies that `@LockedPlanningMethod` allows PrPy to parallelize multiple planners. I think you should r